### PR TITLE
Add in blog post about sbt BSP

### PR DESCRIPTION
**Pull request by ckipp01**
So this is a stab at making sure the BSP thing is clear for users. I did my best to _very_ briefly outline what BSP is, what effect does this have on a Metals user, and how to try the new sbt BSP support if they like. Let me know what you think. Hopefully this can also allow for you to not have to write much about it in the release notes @tgodzik and instead just link to this.

### CODE REVIEW

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (tgodzik) [inline website/blog/2020-11-06-sbt-BSP-support.md:39]: Could you maybe add titles to each icon?

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (tgodzik) [inline website/blog/2020-11-06-sbt-BSP-support.md:67]:
```suggestion
we'll go over down below) that can generate this file for you if it doesn't exist,
```

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (tgodzik) [inline website/blog/2020-11-06-sbt-BSP-support.md:68]:
```suggestion
and then you will be automatically connected to sbt. If this file already exist, then
```

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (tgodzik) [inline website/blog/2020-11-06-sbt-BSP-support.md:69]:
```suggestion
we provide a simple way for you to "switch" build servers and for your choice to
```

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (tgodzik) [inline website/blog/2020-11-06-sbt-BSP-support.md:92]:
```suggestion
- Bloop offers sbt file support that enables completions and hover, whereas this is not
```

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (tgodzik) [inline website/blog/2020-11-06-sbt-BSP-support.md:112]:
```suggestion
If no `.bsp/sbt.json` exists yet, you can generate it with the new
```

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (tgodzik) [inline website/blog/2020-11-06-sbt-BSP-support.md:145]:
```suggestion
pr](https://github.com/scalameta/metals/pull/2154). As always please don't
```

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (tgodzik) [inline website/blog/2020-11-06-sbt-BSP-support.md:146]:
```suggestion
hesitate to ask questions on our various channels, submit issues or create new
```

I did comment on this in a couple of places, but maybe it's my bad knowledge of the English language. Do we need a coma before `and` or `or` ?

**Review by tgodzik — CHANGES_REQUESTED** [review]
Great work @ckipp01 ! It will all be super useful for users :100:

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (ckipp01) [reply website/blog/2020-11-06-sbt-BSP-support.md:5]: Huh, funny enough `https://github.com/ckipp01.png` just redirects to `https://avatars2.githubusercontent.com/u/13974112?v=4`. So I can at least shorten it to that.

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (ckipp01) [reply website/blog/2020-11-06-sbt-BSP-support.md:12]: 👍 ha yes, trying to do my duty as a good board member and bring attention to it.

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (ckipp01) [reply website/blog/2020-11-06-sbt-BSP-support.md:16]: 👍 I'll take the `the`, but replace the `,` with a `:` which is more fitting I think.

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (ckipp01) [reply website/blog/2020-11-06-sbt-BSP-support.md:19]: Ah, I was sort of thinking they would be so close together it didn't matter, but no worries. I changed it to `upcoming 0.9.5 release`

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (ckipp01) [reply website/blog/2020-11-06-sbt-BSP-support.md:146]: Welcome to the oxford comma debate. https://www.grammarly.com/blog/oxford-comma-debate/

In short, both are correct 😆

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (ckipp01) [reply website/blog/2020-11-06-sbt-BSP-support.md:39]: Yea sure, I added them in before each image.

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (eloots) [inline website/blog/2020-11-06-sbt-BSP-support.md:50]:
```suggestion
at the moment. When you open a fresh project, you're prompted to import your
```

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (eloots) [inline website/blog/2020-11-06-sbt-BSP-support.md:53]:
```suggestion
continue to edit, diagnostics and other information are flowing back and forth
```

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (ckipp01) [reply website/blog/2020-11-06-sbt-BSP-support.md]: Actually thinking about this a bit more, these instructions mention the new `generate-bsp-config` which users won't have until the new release of the VS Code extension, so maybe we shouldn't advertise this too much before the actual release to avoid confusion? wdyt?

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (eloots) [inline website/blog/2020-11-06-sbt-BSP-support.md:59]: The question is quoted, so I would add the "mean for Metals" part to it.

```suggestion
Another question you may have is "what does sbt BSP support mean for Metals"?
```

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (tgodzik) [reply website/blog/2020-11-06-sbt-BSP-support.md]: The release notes will come up only after the VS Code extension is also released, so it should be fine.

Comment (tgodzik) [reply website/blog/2020-11-06-sbt-BSP-support.md]: We can also try to maybe publish it at the same time then? No need for the upcoming in that case.

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (eloots) [inline website/blog/2020-11-06-sbt-BSP-support.md:60]:
```suggestion
This means a couple different things. Up until this point, if you wanted to use
```

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (eloots) [inline website/blog/2020-11-06-sbt-BSP-support.md:78]:
```suggestion
At this point you may be asking, "what's the difference?". For an average user,
```

**Review by eloots — COMMENTED** [review]
Left some minor comments and suggestions. Looks good!

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (ckipp01) [reply website/blog/2020-11-06-sbt-BSP-support.md]: Sounds good 👍

**Comment by ckipp01** [comment]
Thanks for the reviews @tgodzik and @eloots!

@tgodzik I think we should wait until just before the release to merge this. This we can reference it in the release notes and link it. That will make sure people don't get confused with the missing `generate-bsp-config` command if they want to try it out before the release.

File: website/blog/2020-11-06-sbt-BSP-support.md
Comment (tgodzik) [reply website/blog/2020-11-06-sbt-BSP-support.md:149]: Och no... :(

**Review by tgodzik — APPROVED** [review]
Just a super minor nitpick. Again and as always, great work @ckipp01 !

**Comment by tgodzik** [comment]
Merging this in preparation for the release. Thanks again @ckipp01 !

**Pull request merged**
